test/battle/ability/aerilate.c

```diff
@@ -21,7 +21,30 @@ SINGLE_BATTLE_TEST("Aerilate turns a Normal-type move into Flying-type move")
     }
 }
 
-TO_DO_BATTLE_TEST("Aerilate can not turn certain moves into Flying type moves");
+SINGLE_BATTLE_TEST("Aerilate can not turn certain moves into Flying type moves")
+{
+    u32 move;
+    PARAMETRIZE { move = MOVE_WEATHER_BALL; }
+    // PARAMETRIZE { move = MOVE_NATURAL_GIFT; } can't hold berry
```

Comment:
Just one suggestion. Everything else lgtm :)

Reply:
That's a pain, I'd have to make another test

Reply:
Alright, then let's just do this:
```suggestion
    // PARAMETRIZE { move = MOVE_NATURAL_GIFT; } TODO: handle this case via Skill Swap
```